src/server/pfs/server/driver.go

```diff
@@ -1799,6 +1799,11 @@ func validatePath(path string) error {
 	if !match {
 		return fmt.Errorf("path (%v) invalid: only printable ASCII characters allowed", path)
 	}
+
+	if isGlob(path) {
+		return fmt.Errorf("path (%v) invalid: globbing characters not allowed in path", path)
```

Comment:
Could we make it so this shows what matched in the regex so they have an idea what needs to change.

Reply:
Yes.